### PR TITLE
First draft of c.rh.c

### PR DESCRIPTION
**Pull request by ryelo**
Getting a 500 from the API and not getting the right navigation

### CODE REVIEW

**Comment by ryelo** [comment]
@karelhala Can you recheck the Router basename just to double check?

**Comment by karelhala** [comment]
Yes the basename is correct.

**Comment by ryelo** [comment]
@jharting I'm going to let you review and then we can merge it

File: src/Routes.js [hunk 1]
```diff
@@ -18,12 +18,42 @@ InsightsRoute.propTypes = {
...
+        }
+    );
+}
+
+export const Routes = ({ childProps: { history }}) => {
+
+    const pathName = window.location.pathname.split('/');
+
+    if (!checkPaths(pathName[1] === 'beta' ? pathName[2] : pathName[1])) {
+        history.push(routes.table);
```
Comment (jharting) [inline src/Routes.js]: what's `routes.table`?

**Pull request merged**